sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
   - Added `SharedTokenCacheCredentialOptions` class with properties `TenantId` and `Username`
   - Added constructor overload to `SharedTokenCacheCredential` which accepts `SharedTokenCacheCredentialOptions` 
   - Added property `SharedTokenCacheTenantId` to `DefaultAzureCredentialOptions`
+- Support for personal account authentication in `DefaultAzureCredential`, `InteractiveBrowserCredential`, and `SharedTokenCacheCredential`
```

Comment:
For my own edification - How did we accomplish this?  I don't see the changes that allow this.  Is this just because you can set a Tenet ID, and that's what used to accomplish this?  With your changes do I need to do anything in my application to start supporting personal account authentication?

Reply:
These changes don't fix this. It was fixed as part of a merged PR where I forgot to update the changelog. https://github.com/Azure/azure-sdk-for-net/pull/8726

Reply:
With the changes in that pull request users can authenticate using personal accounts with no code changes.

Reply:
Gotcha!